package.json

```diff
@@ -107,9 +107,9 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^4.0.2",
     "lerna": "^3.16.4",
-    "mocha": "^6.2.0",
+    "mocha": "^6.2.1",
     "nyc": "^14.1.1",
-    "prettier": "1.17.0",
+    "prettier": "1.18.2",
```

Comment:
What insight did you gain over https://github.com/mui-org/material-ui/pull/16717?